Add Prathamesh Dabre to Contributors list

praths opened the pull request.

praths committed:
Add Prathamesh Dabre to Contributors list

Roshanjossey commented:
Hi  @praths, I'm quite elated about your pull request. I wanna evolve this project to addresses various problems faced by first-time contributors. I'd love to learn about your journey in open source community, the problems, pain points you had etc.
Could you explain how you felt when you went through the tutorial, made a pull request and learned that I merged it?

 We’ve recently added social share  to our web app. Could you please go to https://roshanjossey.github.io/first-contributions/#social-share and share your first contribution to open source?  Also, check out projects with easy issues while you’re there.

praths commented:
Hi @Roshanjossey, this is a delightful experience and I enjoyed my very first GitHub commit in "Open Source". though I have faced some problems in a journey that I feel important to address. mistakenly I have changed the contributing.MD file and then no way I can get rid of it. So I just deleted the entire local repository and cloned it again and pushed my commit.
Thanks for the nice project you have created for first-time contributors.

Roshanjossey commented:
@praths, I'd suggest you to learn `git checkout` command to undo changes using git.
I’d urge you to continue contributing to other open source projects.
We also have a slack team to help/mentor people trying to contribute to open source projects.
You can join the team by going to https://firstcontributions.herokuapp.com
